### PR TITLE
Mail Improvement 3125

### PR DESCRIPTION
**Pull request by carlangas159**
Add prev and next button into message view.

### CODE REVIEW

**Comment by jmontoyaa** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 6
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5738743&bid=18771964)

File: main/inc/lib/message.lib.php [hunk 1]
```diff
@@ -1472,13 +1493,25 @@ public static function showMessageBox($messageId, $type)
...
-                $message_content .= '<a href="inbox.php?'.$social_link.'">'.
-                    Display::return_icon('back.png', get_lang('ReturnToInbox')).'</a> &nbsp';
-                $message_content .= '<a href="new_message.php?re_id='.$messageId.'&'.$social_link.'">'.
-                    Display::return_icon('message_reply.png', get_lang('ReplyToMessage')).'</a> &nbsp';
-                $message_content .= '<a href="inbox.php?action=deleteone&id='.$messageId.'&'.$social_link.'" >'.
-                    Display::return_icon('delete.png', get_lang('DeleteMessage')).'</a>&nbsp';
-                break;
+                    if ($idNextMessage != 0)
+                        {
+                            $message_content .= '<a title="' . get_lang('ScormNext') . '" href="view_message.php?type=' . $type . '&id=' . $idNextMessage . '" style="float:right;">' .
```
Comment (jmontoyaa) [inline main/inc/lib/message.lib.php:1498]: Codacy found an issue: [Line exceeds maximum limit of 120 characters; contains 183 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5738743)

File: main/inc/lib/message.lib.php [hunk 2]
```diff
@@ -1472,13 +1493,25 @@ public static function showMessageBox($messageId, $type)
...
+                    if ($idPrevMessage != 0)
+                        {
+                            $message_content .= '<a title="' . get_lang('ScormPrevious') . '" href="view_message.php?type=' . $type . '&id=' . $idPrevMessage . '" style="float:right;    padding-right: 15px;">' .
+                                '<em class="fa fa-reply fa-2x"></em></a> &nbsp';
+                        }
+                    $message_content .= '<a href="inbox.php?' . $social_link . '">' .
+                        Display::return_icon('back.png', get_lang('ReturnToInbox')) . '</a> &nbsp';
+                    $message_content .= '<a href="new_message.php?re_id=' . $messageId . '&' . $social_link . '">' .
+                        Display::return_icon('message_reply.png', get_lang('ReplyToMessage')) . '</a> &nbsp';
+                    $message_content .= '<a href="inbox.php?action=deleteone&id=' . $messageId . '&' . $social_link . '" >' .
```
Comment (jmontoyaa) [inline main/inc/lib/message.lib.php:1510]: Codacy found an issue: [Line exceeds maximum limit of 120 characters; contains 125 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5738743)

File: main/inc/lib/message.lib.php [hunk 3]
```diff
@@ -1366,9 +1366,30 @@ public static function showMessageBox($messageId, $type)
         $result = Database::query($query);
         $row = Database::fetch_array($result, 'ASSOC');
 
-        if (empty($row)) {
-            return '';
-        }
+        if (empty($row))
+            {
+                return '';
+            }
```
Comment (jmontoyaa) [inline main/inc/lib/message.lib.php:1372]: Codacy found an issue: [Line indented incorrectly; expected 8 spaces, found 12](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5738743)

File: main/inc/lib/message.lib.php [hunk 4]
```diff
@@ -1472,13 +1493,25 @@ public static function showMessageBox($messageId, $type)
...
                 break;
             case self::MESSAGE_TYPE_INBOX:
-                $message_content .= '<a href="inbox.php?'.$social_link.'">'.
-                    Display::return_icon('back.png', get_lang('ReturnToInbox')).'</a> &nbsp';
-                $message_content .= '<a href="new_message.php?re_id='.$messageId.'&'.$social_link.'">'.
-                    Display::return_icon('message_reply.png', get_lang('ReplyToMessage')).'</a> &nbsp';
-                $message_content .= '<a href="inbox.php?action=deleteone&id='.$messageId.'&'.$social_link.'" >'.
-                    Display::return_icon('delete.png', get_lang('DeleteMessage')).'</a>&nbsp';
-                break;
+                    if ($idNextMessage != 0)
```
Comment (jmontoyaa) [inline main/inc/lib/message.lib.php:1496]: Codacy found an issue: [Line indented incorrectly; expected 16 spaces, found 20](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5738743)

File: main/inc/lib/message.lib.php [hunk 5]
```diff
@@ -1472,13 +1493,25 @@ public static function showMessageBox($messageId, $type)
...
+                        {
+                            $message_content .= '<a title="' . get_lang('ScormNext') . '" href="view_message.php?type=' . $type . '&id=' . $idNextMessage . '" style="float:right;">' .
+                                '<em class="fa fa-share fa-2x"></em></a> &nbsp';
+                        }
+                    if ($idPrevMessage != 0)
```
Comment (jmontoyaa) [inline main/inc/lib/message.lib.php:1501]: Codacy found an issue: [Line indented incorrectly; expected 16 spaces, found 20](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5738743)

File: main/inc/lib/message.lib.php [hunk 6]
```diff
@@ -1472,13 +1493,25 @@ public static function showMessageBox($messageId, $type)
...
+                        {
+                            $message_content .= '<a title="' . get_lang('ScormPrevious') . '" href="view_message.php?type=' . $type . '&id=' . $idPrevMessage . '" style="float:right;    padding-right: 15px;">' .
```
Comment (jmontoyaa) [inline main/inc/lib/message.lib.php:1503]: Codacy found an issue: [Line exceeds maximum limit of 120 characters; contains 211 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5738743)

**Comment by jmontoyaa** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 7
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5738743&bid=18771964)

File: main/inc/lib/message.lib.php (hunk 3 above)
Comment (jmontoyaa) [inline main/inc/lib/message.lib.php:1372]: Codacy found an issue: [Line indented incorrectly; expected 8 spaces, found 12](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5738743)

File: main/inc/lib/message.lib.php [hunk 7]
```diff
@@ -1466,19 +1487,35 @@ public static function showMessageBox($messageId, $type)
...
+
+
+                    $message_content .= '<a href="inbox.php?' . $social_link . '">' .
+                        Display::return_icon('icons/22/arrow_up_1.png', get_lang('ReturnToInbox')) . '</a> &nbsp';
+
+                    $message_content .= '<a href="new_message.php?re_id=' . $messageId . '&' . $social_link . '">' .
+                        Display::return_icon('message_reply.png', get_lang('ReplyToMessage')) . '</a> &nbsp';
+                    $message_content .= '<a href="inbox.php?action=deleteone&id=' . $messageId . '&' . $social_link . '" >' .
+                        Display::return_icon('delete.png', get_lang('DeleteMessage')) . '</a>&nbsp';
+                    if ($idPrevMessage != 0)
```
Comment (jmontoyaa) [inline main/inc/lib/message.lib.php:1505]: Codacy found an issue: [Line indented incorrectly; expected 16 spaces, found 20](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5738743)

File: main/inc/lib/message.lib.php [hunk 8]
```diff
@@ -1466,19 +1487,35 @@ public static function showMessageBox($messageId, $type)
...
+                        {
+                            $message_content .= '<a title="' . get_lang('ScormPrevious') . '" href="view_message.php?type=' . $type . '&id=' . $idPrevMessage . '" ">' .
```
Comment (jmontoyaa) [inline main/inc/lib/message.lib.php:1507]: Codacy found an issue: [Line exceeds maximum limit of 120 characters; contains 168 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5738743)

File: main/inc/lib/message.lib.php [hunk 9]
```diff
@@ -1466,19 +1487,35 @@ public static function showMessageBox($messageId, $type)
...
+                                Display::return_icon('icons/22/back.png', get_lang('ScormPrevious')) . '</a> &nbsp';
+                        }
+                    if ($idNextMessage != 0)
+                        {
+                            //Display::return_icon('back.png', get_lang('ReturnToInbox')) . '</a> &nbsp';
+                            $message_content .= '<a title="' . get_lang('ScormNext') . '" href="view_message.php?type=' . $type . '&id=' . $idNextMessage . '">' .
```
Comment (jmontoyaa) [inline main/inc/lib/message.lib.php:1513]: Codacy found an issue: [Line exceeds maximum limit of 120 characters; contains 162 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5738743)

File: main/inc/lib/message.lib.php [hunk 10]
```diff
@@ -1466,19 +1487,35 @@ public static function showMessageBox($messageId, $type)
...
         switch ($type) {
             case self::MESSAGE_TYPE_OUTBOX:
-                $message_content .= '<a href="outbox.php?'.$social_link.'">'.
-                    Display::return_icon('back.png', get_lang('ReturnToOutbox')).'</a> &nbsp';
-                $message_content .= '<a href="outbox.php?action=deleteone&id='.$messageId.'&'.$social_link.'" >'.
-                    Display::return_icon('delete.png', get_lang('DeleteMessage')).'</a>&nbsp';
-                break;
+                    $message_content .= '<a href="outbox.php?' . $social_link . '">' .
+                        Display::return_icon('back.png', get_lang('ReturnToOutbox')) . '</a> &nbsp';
+                    $message_content .= '<a href="outbox.php?action=deleteone&id=' . $messageId . '&' . $social_link . '" >' .
```
Comment (jmontoyaa) [inline main/inc/lib/message.lib.php:1492]: Codacy found an issue: [Line exceeds maximum limit of 120 characters; contains 126 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5738743)

File: main/inc/lib/message.lib.php [hunk 11]
```diff
@@ -1466,19 +1487,35 @@ public static function showMessageBox($messageId, $type)
...
-                    Display::return_icon('delete.png', get_lang('DeleteMessage')).'</a>&nbsp';
-                break;
+
+
+                    $message_content .= '<a href="inbox.php?' . $social_link . '">' .
+                        Display::return_icon('icons/22/arrow_up_1.png', get_lang('ReturnToInbox')) . '</a> &nbsp';
+
+                    $message_content .= '<a href="new_message.php?re_id=' . $messageId . '&' . $social_link . '">' .
+                        Display::return_icon('message_reply.png', get_lang('ReplyToMessage')) . '</a> &nbsp';
+                    $message_content .= '<a href="inbox.php?action=deleteone&id=' . $messageId . '&' . $social_link . '" >' .
```
Comment (jmontoyaa) [inline main/inc/lib/message.lib.php:1503]: Codacy found an issue: [Line exceeds maximum limit of 120 characters; contains 125 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5738743)

File: main/inc/lib/message.lib.php [hunk 12]
```diff
@@ -1466,19 +1487,35 @@ public static function showMessageBox($messageId, $type)
...
+                        Display::return_icon('delete.png', get_lang('DeleteMessage')) . '</a>&nbsp';
+                    if ($idPrevMessage != 0)
+                        {
+                            $message_content .= '<a title="' . get_lang('ScormPrevious') . '" href="view_message.php?type=' . $type . '&id=' . $idPrevMessage . '" ">' .
+                                Display::return_icon('icons/22/back.png', get_lang('ScormPrevious')) . '</a> &nbsp';
+                        }
+                    if ($idNextMessage != 0)
```
Comment (jmontoyaa) [inline main/inc/lib/message.lib.php:1510]: Codacy found an issue: [Line indented incorrectly; expected 16 spaces, found 20](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5738743)

**Comment by jmontoyaa** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5738743&bid=18771964)

File: main/inc/lib/message.lib.php [hunk 13]
```diff
@@ -1466,19 +1486,31 @@ public static function showMessageBox($messageId, $type)
...
+
+                    $message_content .= '<a href="new_message.php?re_id='.$messageId.'&'.$social_link.'">'.
+                        Display::return_icon('message_reply.png', get_lang('ReplyToMessage')).'</a> &nbsp';
+                    $message_content .= '<a href="inbox.php?action=deleteone&id='.$messageId.'&'.$social_link.'" >'.
+                        Display::return_icon('delete.png', get_lang('DeleteMessage')).'</a>&nbsp';
+                    if ($idPrevMessage != 0) {
+                        $message_content .= '<a title="'.get_lang('ScormPrevious').'" href="view_message.php?type='.$type.'&id='.$idPrevMessage.'" ">'.
+                                Display::return_icon('icons/22/back.png', get_lang('ScormPrevious')).'</a> &nbsp';
+                    }
+                    if ($idNextMessage != 0) {
```
Comment (jmontoyaa) [inline main/inc/lib/message.lib.php:1507]: Codacy found an issue: [Line indented incorrectly; expected 16 spaces, found 20](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5738743)

File: main/inc/lib/message.lib.php [hunk 14]
```diff
@@ -1466,19 +1486,31 @@ public static function showMessageBox($messageId, $type)
...
-                break;
+
+                    $message_content .= '<a href="inbox.php?'.$social_link.'">'.
+                        Display::return_icon('icons/22/arrow_up_1.png', get_lang('ReturnToInbox')).'</a> &nbsp';
+
+                    $message_content .= '<a href="new_message.php?re_id='.$messageId.'&'.$social_link.'">'.
+                        Display::return_icon('message_reply.png', get_lang('ReplyToMessage')).'</a> &nbsp';
+                    $message_content .= '<a href="inbox.php?action=deleteone&id='.$messageId.'&'.$social_link.'" >'.
+                        Display::return_icon('delete.png', get_lang('DeleteMessage')).'</a>&nbsp';
+                    if ($idPrevMessage != 0) {
```
Comment (jmontoyaa) [inline main/inc/lib/message.lib.php:1503]: Codacy found an issue: [Line indented incorrectly; expected 16 spaces, found 20](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5738743)

File: main/inc/lib/message.lib.php [hunk 15]
```diff
@@ -1466,19 +1486,31 @@ public static function showMessageBox($messageId, $type)
...
+                        $message_content .= '<a title="'.get_lang('ScormPrevious').'" href="view_message.php?type='.$type.'&id='.$idPrevMessage.'" ">'.
+                                Display::return_icon('icons/22/back.png', get_lang('ScormPrevious')).'</a> &nbsp';
+                    }
```
Comment (jmontoyaa) [inline main/inc/lib/message.lib.php:1506]: Codacy found an issue: [Line indented incorrectly; expected 8 spaces, found 12](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5738743)

File: main/inc/lib/message.lib.php [hunk 16]
```diff
@@ -1466,19 +1486,31 @@ public static function showMessageBox($messageId, $type)
...
+
+                    $message_content .= '<a href="inbox.php?'.$social_link.'">'.
+                        Display::return_icon('icons/22/arrow_up_1.png', get_lang('ReturnToInbox')).'</a> &nbsp';
+
+                    $message_content .= '<a href="new_message.php?re_id='.$messageId.'&'.$social_link.'">'.
+                        Display::return_icon('message_reply.png', get_lang('ReplyToMessage')).'</a> &nbsp';
+                    $message_content .= '<a href="inbox.php?action=deleteone&id='.$messageId.'&'.$social_link.'" >'.
+                        Display::return_icon('delete.png', get_lang('DeleteMessage')).'</a>&nbsp';
+                    if ($idPrevMessage != 0) {
+                        $message_content .= '<a title="'.get_lang('ScormPrevious').'" href="view_message.php?type='.$type.'&id='.$idPrevMessage.'" ">'.
```
Comment (jmontoyaa) [inline main/inc/lib/message.lib.php:1504]: Codacy found an issue: [Line exceeds maximum limit of 120 characters; contains 168 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5738743)

File: main/inc/lib/message.lib.php [hunk 17]
```diff
@@ -1466,19 +1486,31 @@ public static function showMessageBox($messageId, $type)
...
+                                Display::return_icon('icons/22/back.png', get_lang('ScormPrevious')).'</a> &nbsp';
+                    }
+                    if ($idNextMessage != 0) {
+                        //Display::return_icon('back.png', get_lang('ReturnToInbox')) . '</a> &nbsp';
+                        $message_content .= '<a title="'.get_lang('ScormNext').'" href="view_message.php?type='.$type.'&id='.$idNextMessage.'">'.
```
Comment (jmontoyaa) [inline main/inc/lib/message.lib.php:1509]: Codacy found an issue: [Line exceeds maximum limit of 120 characters; contains 162 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5738743)

**Comment by ywarnier** [comment]
This applies to issue #3125

**Pull request merged**